### ISSUE TITLE
proseed for fs25?

### ISSUE THREAD

**Issue by Peterbc83** [stijnwop/proSeed#58]
do we get to see this great mod for farming simulator 25. i miss it already.

**Comment by blue-painted** [comment]
I would also like to see proseed for FS25 please!

**Comment by Justice-Molton** [comment]
Any chance to get in 25?

**Comment by chrissnave** [comment]
They just added "tramlines" as part of the FS25 Precision Farming DLC & this makes me want Proseed in FS25 even more. The Precision Farming tramlines are no substitute for this excellent mod. I really miss Proseed.

**Comment by Jannes236** [comment]
Yeah, me also! The Tramlimes of FS are horrible! And I would also bee really happy, if the guidance steering comes back to FS25!

**Comment by FS22fanCatholic** [comment]
I too miss proseed, it was a great mod. I also like others have said I miss guidance steering as the base game one isn't as good as the one we had in FS22. Both really great mods. I hope you will convert them sometime. Thanks God bless all ✝️🙏✝️

**Comment by chrissnave** [comment]
If the author is not planning to maintain & update this mod could he please change the license so that Proseed can be forked over to someone who will continue it? It's such a shame to see such a great mod get left behind.

**Comment by stijnwop** [comment]
It is being worked on. Everything due time.

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/ff75f95f-dffb-468c-999d-81e4f8b4058b" />

**Comment by chrissnave** [comment]
That is fantastic news. You just made my day!

**Comment by Jannes236** [comment]
Mine also! Best news since a long time! 😍

**Comment by Omnissias** [comment]
That is -the- one mod I still really miss. The tram lines of Precision farming are just horrible and hilarious. Especially since often they dont even connect the field entrance or have angles above 90 degree.

Can't wait for this one to come out.

**Comment by chrissnave** [comment]
I'm excited to hear about any progress on Proseed FS25.

Or if this is not happening, might I suggest contacting Kim Brandwijk from Paint-a-Farm Modding. Kim is actively developing the Extended Precision Farming mod. Maybe integrating Proseed into that might be an option?

[Facebook: Paint-a-Farm Modding](https://www.facebook.com/PaintFarm)

**Comment by o0bonde** [comment]
found a post on the danish facebookpage that one have convertet proseed from fs22 to fs25

<img width="632" height="498" alt="Image" src="https://github.com/user-attachments/assets/9b89d111-5f4a-4e1a-99a4-3f1e15651ab2" />

i think it is bullshit

**Comment by chrissnave** [comment]
I would really like to use Proseed but I'm not going to risk downloading that. Looks sketchy. I'd feel much safer running the official version.

Any progress Stijn? Its been 4 months since you posted that screen shot of Proseed working in FS25.

**Comment by DevCookiie** [comment]
Is there any status on how far you have come with ProSeed as it is a bit lacking now so that you can make some better spear tracks?

**Comment by fdecoux-commits** [comment]
Quand Est ce que le mod proseed fs25 pourrait il sortir ?

**Comment by tytandev12-crypto** [comment]
HI @stijnwop Hi, my name is Michał. I’m a beginner FS modder; I’ve been converting existing mods and have improved your ProSeed mod. I wanted to ask if I could submit it to ModHub, or if that’s not really an option. If not, is there a chance it could be accepted given the changes I made to the code? I already have the mod ready; if you'd like to see it, we can get in touch. my e-mail: tytan.dev12@gmail.com
